app/src/main/java/de/danoeh/antennapod/ui/screen/playback/SleepTimerDialog.java

```diff
@@ -40,8 +51,65 @@ public class SleepTimerDialog extends DialogFragment {
     private LinearLayout timeSetup;
     private LinearLayout timeDisplay;
     private TextView time;
+    private Spinner sleepTimerType;
     private CheckBox chAutoEnable;
 
+    Button extendSleepFiveMinutesButton;
+    Button extendSleepTenMinutesButton;
+    Button extendSleepTwentyMinutesButton;
```

Comment:
This seems like a good opportunity to migrate the dialog to ViewBindings. See for example `QuickFeedDiscoveryFragment`

Reply:
I'd keep this for a later PR if that's ok with you, in case I mess up something and it will just add to the changes in this PR